frontend/src/components/widgets/inputs/CheckboxInput.svelte

```diff
@@ -47,7 +47,7 @@
 <style lang="scss" global>
 	.checkbox-input {
 		flex: 0 0 auto;
-		align-items: center;
+		align-items: right;
```

Comment:
This change appears to move the checkbox in a strange way. Is it needed?

Reply:
Reverted, had forgotten to undo this.